### ISSUE TITLE
NLTK parsing of empty/1-item groups gives bad LTL

### ISSUE THREAD

**Issue by cfinucane** [VerifiableRobotics/LTLMoP#45]
For example, try:

```
Group emptygroup is empty
Group oneitemgroup is radio

Visit each emptygroup
Always do any oneitemgroup
```

This returns:

```
[]((s.radio | )) & 
([]<>(empty) & ) & ...
```

**Comment by cfinucane** [comment]
Note that this is as of 1c0aef7defecdb25a3

**Comment by cfinucane** [comment]
Should be fixed as of 7623723a8285a44baba9
